src/plugins/lua/force_actions.lua

```diff
@@ -63,6 +64,66 @@ local function gen_cb(expr, act, pool, message, subject, raction, honor, limit,
 
   return function(task)
 
+    local function fill_vars(repl, var_class, var_payload)
+      -- fill template vars prefixed with 
+      --  'selector::' => fill with value extracted by a selector
+      --  'symbol::'   => fill with matching symbol option string(s)
+      --  'task::'     => fill with matching task attribute
+      --                   (only queue_id and uid allowed)
+
+      if var_class == "selector" then
+        local selector = lua_selectors.create_selector_closure(rspamd_config, var_payload, '', true)
+        if not selector then 
+          rspamd_logger.errx(rspamd_config, 'could not create selector [%1]', var_payload)
+          return "((could not create selector))"
+        end
+        local extracted = selector(task)
+        if extracted then
+          -- replace non ascii chars
+          if type(extracted) == 'table' then
+            extracted = table.concat(extracted, ',')
+          end
+          extracted = string.gsub(extracted, '[\128-\255]', '?')
+        else
+          rspamd_logger.errx(rspamd_config, 'could not extract value with selector [%1]', var_payload)
+          extracted = '((error extracting value))'
+        end
+        return extracted
+      end
+
+      if var_class == "symbol" then
+        local symb_opts
+        local symb_strs = {}
+        if task:has_symbol(var_payload) then
+          local symb_tbl = task:get_symbol(var_payload)
+          for _,symb in ipairs(symb_tbl) do
+            local symb_opts = symb.options
+            if symb_opts then
+              -- replace non ascii chars
+              symb_strs[#symb_strs+1] = string.gsub(table.concat(symb_opts, ','), '[\128-\255]', '?')
```

Comment:
Same here, why this is needed?

Reply:
Same as above. Symbol option strings could contain utf-8 chars as well. (They certainly do for the custom lua plugin we use, that adds blocked filenames to symbol option strings)